### ISSUE TITLE
Can't use the mouse to play a song

### ISSUE THREAD

**Issue by conchesness** [okonek/tidal-cli-client#15]
Thank you for providing this package.  I hope to get it running soon!  My system is as follows:

Ubuntu 16.04
OpenJDK 8
W3M 0.5.3
W3M-img 0.5.3
MPV 0.14.0
NODEJS 8.10
tidal-cli-client@1.2.8

Everything works reasonably well but I can't play a song once I get a list.  I can hit 'n' to queue a song but I can't use the mouse to click on a song for it to play.  W3M mouse functions work fine when I run w3m and browse a website.

**Comment by okonek** [comment]
Yes, you can't use your mouse to click in tidal-cli-client. I thought it would be unnecessary, but I can add it.
